lib/node_modules/@stdlib/math/base/tools/normhermitepolyf/test/fixtures/python/runner.py

```diff
@@ -73,29 +73,29 @@ def main():
     """Generate fixture data."""
     # Random values across `n` and `x`:
     n = np.random.randint(1, 20, 100)
-    x = np.float16(np.random.random(100)*50.0)
+    x = np.float32(np.random.random(100)*50.0)
     gen(n, x, "random2.json")
 
     # Medium negative:
-    x = np.linspace(-709.78, -1.0, 1000, dtype=np.float16)
+    x = np.linspace(-709.78, -1.0, 1000, dtype=np.float32)
     gen(1, x, "medium_negative_1.json")
     gen(2, x, "medium_negative_2.json")
     gen(5, x, "medium_negative_5.json")
 
     # Medium positive:
-    x = np.linspace(1.0, 709.78, 1000, dtype=np.float16)
+    x = np.linspace(1.0, 709.78, 1000, dtype=np.float32)
     gen(1, x, "medium_positive_1.json")
     gen(2, x, "medium_positive_2.json")
     gen(5, x, "medium_positive_5.json")
 
     # Small positive:
```

Comment:
Here `Small positive` and `Small negetive` if I use `1.0` and `float32`, it will bring about %1 test failures.